Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H

```diff
@@ -80,32 +80,21 @@ class FiniteDifferenceSolver
 
 #ifdef WARPX_MAG_LLG
         /**
-          * \brief Macroscopic M-update for non-vacuum medium using finite-difference algorithm
+          * \brief Macroscopic M-update, H-update and B-update for non-vacuum medium using finite-difference algorithm
```

Comment:
```suggestion
          * \brief Macroscopic M-update, H-update and B=mu_o(H+M) computation for non-vacuum medium using finite-difference algorithm
```

Reply:
just for clarity that we dont "update" B using curl of E, unlike the E-B update

Reply:
yeah!